src/modules/Inquiries.js

```diff
@@ -8,13 +9,16 @@ const Inquiries = {
       return
     }, 500)
     try {
+
       let params = { inquiry: formData }
       let response = await axios.post('/inquiries', params)
       store.dispatch({
         type: 'SET_SUBMIT_MESSAGE',
         payload: response.data.message,
-      })
+      })      
+      ahoy.track(`answer`, question: 'submit');
     } catch (error) {}
+
```

Comment:
`{question: 'submit'}` should be an object :D

Reply:
ok